lib/Sema/ImportResolution.cpp

```diff
@@ -367,14 +347,16 @@ void ImportResolver::addImport(const UnboundImport &I, ModuleDecl *M) {
 // MARK: Import module loading
 //===----------------------------------------------------------------------===//
 
-ModuleDecl *
-ImportResolver::getModule(ImportPath::Module modulePath) {
+static ModuleDecl *
+getModuleImpl(ImportPath::Module modulePath, ModuleDecl *loadingModule,
+              bool canImportBuiltin) {
```

Comment:
With the removal of `forceImplicitImportLoading`, do we still need the separation between `getModuleImpl` and `getModule`?

Reply:
We don't strictly need it, but I think the code is a bit cleaner with this logic disentangled from `UnboundImport`, so why re-entangle it?

Reply:
Fair enough!